docs/repositories-coverage/commits.md

```diff
@@ -0,0 +1,78 @@
+---
+page_name: commit
+file_name: commits
+product: coverage
+---
+
+# Coverage Commits page
+
+The **Coverage Commits page** displays an overview of the commits in your repository, such as the analysis status and coverage for each commit. This allows you to monitor the evolution of coverage in your repository per commit.
+
+By default, the page lists the commits on the main branch of your repository but if you have [more than one branch enabled](../repositories-configure/managing-branches.md) you can use the drop-down list at the top of the page to display commits on other branches.
+
+![Commits page](images/commits.png)
+
+Click a specific commit to see detailed information about the code coverage changes introduced by that commit.
+
+![Commit detail](images/commits-detail.png)
+
+The next sections describe each area of the commit detail page.
+
+## Commit information {: id="info"}
+
+This area displays detailed information about the commit:
+
+-   Commit message
+-   Committer, SHA hash, and parent commits
+-   Date
+-   Link to the commit on your Git provider
+
+![Commit information](images/commits-detail-information.png)
+
+<!--coverage-overview-start-->
+## {{ page.meta.page_name.capitalize() }} coverage overview {: id="coverage-overview"}
+
+This area displays the coverage gate status and an overview of the coverage metrics for the {{ page.meta.page_name }}:
+
+-   The quality gate status is either **Up to coverage standards** or **Not up to coverage standards** depending on the [coverage gate rules](../repositories-configure/adjusting-quality-gates.md) for your repository.
+
+    If you don't have any rules enabled for {{ page.meta.page_name }}s, the status is always **Up to coverage standards**.
+
+-   The following coverage metrics for the {{ page.meta.page_name }}, displayed either as a **positive or negative variation**, {% if page.meta.page_name == "commit" %}or **no variation** (represented by `=`){% else %}**no variation** (represented by `=`), or **not applicable** (represented by `∅`){% endif %}:
+
+{% if page.meta.page_name == "commit" %}
+    -   **Coverage variation:** Variation of code coverage percentage relative to the parent commit
+    -   **Total coverage:** Coverage value for the current commit
```

Comment:
I’m wondering if this is clear and easy to understand. When we say that Total Coverage is the coverage for the current commit, will some users think that this is the Coverage for the changes in the commit?

Reply:
Good question. While I was creating the page I noticed that this is a new metric that we're displaying on the commit details page ([it's not there on the Codacy Quality page](https://docs.codacy.com/repositories/commits/#quality-overview)).

"Intuitively" I assumed that it was the total coverage for the repository on that commit (since a commit is a snapshot of all files on the repository). If this metric was the coverage of just the changes introduced by the commit, it should be called "diff coverage" - also to keep things consistent between commits and pull requests. :thinking: 

Anyway, on the docs side we can already clarify this better. Here's a proposal:

```suggestion
    -   **Total coverage:** Coverage value of the repository at this commit
```